packages/react-widgets/src/widgets/GeocoderWidget.js

```diff
@@ -106,10 +111,13 @@ function GeocoderWidget(props) {
         setIsLoading(true);
         const result = await geocodeStreetPoint(credentials, {
           searchText,
+          method: props.method,
```

Comment:
Does the developer needs the set the method prop in the widget declaration?

I mean, do you need to do something like this?:

<GeocoderWidget method='lds'>

If possible, we should come up with a way that works without having to specify the method explicitly. I don't know how the solution could look like but we have an environment variable that we can check to see if we have a CARTO 2 or CARTO 3 application.

Reply:
@borja-munoz 
This was only proposition, as i thought it would good to be explicit.
In general, i agree so i've already removed this props from last patch.

As for now, we always 
 - use LDS for V3 credentials
 - use SQL for V2 credentials (old not changed behavior)

Reply:
one param less to document, 👍🏻